pkg/resources/manifests/deploy.yaml

```diff
@@ -117,6 +117,12 @@ spec:
             exec:
               command:
                 - "/cleanup.sh"
+        readinessProbe:
+          periodSeconds: 5
```

Comment:
nit: I wouldn't specify any value unless we want this to be more aggressive than defaults for some reason (default is 10s, see: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#Probe).

Also, this setting were kind of broken until recently (not sure they are completely fixed yet, maybe progressing to GA soon or something). If you really want this to be 5, we can search if there is some feature gate to recommend to enable or what.

But maybe this is all just personal taste, so feel free to ignore :)

Reply:
I'd like this to be a bit more aggressive because it determines the time a user has to wait when deploying it. Do you think having 5 there is such a big issue? (I mean, could we break something with that number?)

Reply:
No, not at all. This sounds good then, sorry for the noise :)

Reply:
Thanks for checking this up!